keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -887,6 +925,14 @@ enhancement:
   CRI or CNI may require updating that component before the kubelet.
 -->
 
+Same as above: all the components must have support for it to work.
+Specifically, kube-apiserver will filter this field if it doesn't recognize/support it.
+If the kubelet doesn't support the field, it will not pull the image, and the path will not be present for the volume manager
+to mount it in, and the pod will fail.
+if the CRI implementation doesn't support it, then the PullImage request will fail and the pod creation will fail.
+
+TODO: check what happens when kubelet is too old to support the feature
```

Comment:
The sig probably needs this information to approve enabling this feature in production clusters in 1.32.